src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -171,6 +171,11 @@ public static class RSessionEvaluationCommands {
             return evaluation.EvaluateAsync<JArray>(script, REvaluationKind.Normal);
         }
 
+        public static Task<JArray> UserLibraryPathsAsync(this IRExpressionEvaluator evaluation) {
+            var script = @"rtvs:::packages.userlibpaths()";
+            return evaluation.EvaluateAsync<JArray>(script, REvaluationKind.Normal);
```

Comment:
I believe you can just do `EvaluateAsync<string[]>`, so long as all values in the array are actually guaranteed to be strings (or null).